slash/core/result.py

```diff
@@ -227,7 +227,7 @@ def _add_error(self, error_list, error=None, frame_correction=0, exc_info=None,
             if is_failure:
                 # force the error object to be marked as failure
                 error.mark_as_failure()
-            _logger.debug('Error added: {0}\n{0.traceback}', error, extra={'to_error_log': 1})
+            _logger.debug('Error added: {0}\n{0.traceback}', error, extra={'highlights': 1})
```

Comment:
I prefer `highlight`, not `highlights` because the log itself is a highlight (singular)